src/Components/Album.jsx

```diff
@@ -1,11 +1,37 @@
-//import React from 'react';
-//import { AlbumName } from "./AlbumName";
+import React from "react";
```

Comment:
with the newest verions of react, this code should not be necessary to have in every component as far as I understood